python/tvm/testing/autotvm.py

```diff
@@ -62,9 +62,12 @@ def matmul(N, L, M, dtype):
 
     # schedule according to config
     yo, yi = cfg["tile_y"].apply(s, C, y)
-    xo, xi = cfg["tile_x"].apply(s, C, x)
+    if cfg["tile_x"].size[-1] > 1:
+        xo, xi = cfg["tile_x"].apply(s, C, x)
```

Comment:
Can you give some context on this in the code? Is this to hit the previously failing case?

Reply:
Yeah, this is to hit the failing test case. It will conditionally split one of the loops so that some configurations have n itervars and some have n+1.

Reply:
Can you add a comment

Reply:
done